mqtt/mqtt.js

```diff
@@ -135,8 +148,14 @@ aedes.authorizePublish = function (client, packet, publish) {
 
 aedes.authorizeSubscribe = function (client, packet, subscribe) {
     // Topics are in the form `channels/<channel_id>/messages`
-    var channel = /^channels\/(.+?)\/messages$/.exec(packet.topic),
-        channelId = channel[1],
+    // Subtopic's are in the form `channels/<channel_id>/messages/<subtopic>`
+    var channel = /^channels\/(.+?)\/messages\/?(.+?)?$/.exec(packet.topic);
```

Comment:
Can this be simplified with `var channel = /^channels\/(.+?)\/messages\/?.*$/.exec(packet.topic);`?

Reply:
We'll see this subsequantly by tests, we need to merge this PR ASAP